src/commands/commit/withClient/index.js

```diff
@@ -26,18 +26,27 @@ const withClient = async (answers: Answers) => {
 
     if (configurationVault.getAutoAdd()) await execa('git', ['add', '.'])
 
-    const { stdout } = await execa('git', [
+    const extraMessageArgs = answers.message
+      ? ['-m', `'${answers.message}'`]
+      : []
+
+    const args = [
       'commit',
       ...isSigned,
       '-m',
-      title,
-      '-m',
-      answers.message
-    ])
+      `'${title}'`,
+      ...extraMessageArgs
+    ]
 
-    console.log(stdout)
+    await execa('git', args, { stdio: 'inherit', shell: true })
```

Comment:
I don't understand why you didn't just add the option argument but go the extra way to not add the extra '-m' if no commit message is passed ?

Reply:
i think "shell:true" produced an error with the previous implementation